tools/backfill_agent.py

```diff
@@ -170,14 +170,18 @@ def analyze_csv(self):
 
     def list_recover_data(self):
         # There must be at least 6 points each day
-        ref_count = 6
-        for s in task_schedule:
-            for b in BROWSER_SET:
-                __count = self.dot_count[b][s]
-                missing = ref_count - __count
+        target_count = 6
+        for case in task_schedule:
+            for browser in BROWSER_SET:
+                case_value_count = self.dot_count[browser][case]
+                missing = max((target_count - case_value_count), 0)
                 if missing > 0:
```

Comment:
this is redundant condition since variable `missing` already greater or equal to 0